test/suites/host-ip-change.robot

```diff
@@ -0,0 +1,229 @@
+*** Settings ***
+Documentation       Host IP and Certificate change test Suite
+
+Resource            ../resources/common.resource
+Resource            ../resources/systemd.resource
+Resource            ../resources/microshift-config.resource
+Resource            ../resources/microshift-process.resource
+
+Library             OperatingSystem
+Library             SSHLibrary
+Library             BuiltIn
+Library             String
+Library             random
+
+Suite Setup         Setup Suite
+Suite Teardown      Teardown Suite
+
+Test Tags           restart    slow
+
+*** Variables ***
+${HOSTS_FILE_PATH}  /etc/hosts
+${NETWORK_FILE_PATH}  /etc/NetworkManager/system-connections/ens3.nmconnection
```

Comment:
This is a wrong assumption that the device `ens3` would always exist.
The device name and number may be different.
```
$ find /etc/NetworkManager/system-connections/ -name \*.nmconnection
/etc/NetworkManager/system-connections/enp1s0.nmconnection
```

Reply:
Updated